### PR TITLE
Added %temp% updates for GCCH and DoD

### PR DESCRIPTION
**Pull request by MSFT-GRRojas**
This was discovered on ICM (https://portal.microsofticm.com/imp/v3/incidents/details/374179487/home)

### CODE REVIEW

File: migration/mm-gov-cloud.md [hunk 1]
```diff
@@ -50,7 +50,8 @@ If your are either a **GCC high** or **DoD** customer, you need to make a change
 1. Download the agent setup file.
 2. Open the setup file and remain on the **Welcome** page. 
 3. Open *C:\Users\<user>\AppData\Roaming\Microsoft\SPMigration\Bin\microsoft.sharepoint.migration.common.dll.config*.
-4. Change the value of *SPOEnvironmentType* from 0 to 2.
+4. Open *C:\Users\<user>\AppData\Local\Temp\SPMigrationAgentSetup\SPMigrationAgentSetup\microsoft.sharepoint.migration.common.dll.config*.
+5. For GCCH, change the value of *SPOEnvironmentType* from 0 to 2; and for DoD change the value of *SPOEnvironmentType* from 0 to 3.
```
Comment (JohanFreelancer9) [inline migration/mm-gov-cloud.md:54]:
```suggestion
5. For GCCH, change the value of *SPOEnvironmentType* from 0 to 2, and for DoD, change the value of *SPOEnvironmentType* from 0 to 3.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @MSFT-GRRojas, kindly commit the minor suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by MSFT-GRRojas — COMMENTED** [review]
Thank you

**Comment by scanum** [comment]
@JoanneHendrickson 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by MSFT-GRRojas — COMMENTED** [review]
Do we know when these will be pushed live?

**Comment by scanum** [comment]
@tiburd Hope all is well. Could you please help with this PR? Thank you.

**Pull request merged**